packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -88,8 +99,7 @@ export const ApplyHandler = withConfig<BaseSqlExecuteArgs>(
 			return;
 		}
 
-		const isInteractive = process.stdout.isTTY;
-		if (isInteractive) {
+		if (isInteractive()) {
```

Comment:
In most other places we use the check `isInteractive() && !CI.isCI()`. I'm working on adding that check into the `confirm` function, so this isn't a blocker for this PR, but just wanted to note it.